### DISCUSSION TITLE
API Limit?

### DISCUSSION THREAD

**Discussion by Buono-Nick** [metafloor/bwip-js#348] [Q&A] [answered]
Hi, just out of curiosity is there a limit to the number of callouts that can be made in a single day? I'm trying to generate barcodes for an entire catalog of items.

**Comment by metafloor** [comment, marked as answer]
There is no hard-coded limit, but please don't slam the service.  It is shared with many other users and runs as a micro service on heroku, with only a single (virtual) cpu.

**Reply by Buono-Nick** [reply]
thank you! I appreciate the quick response
